components/brave_ads/core/internal/ml/transformation/mapped_tokens_transformation_flat.cc

```diff
@@ -0,0 +1,104 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Why is this code duplicated? Or do you just need to rename this file to `mapped_tokens_transaction.cc`